css/styles.css

```diff
@@ -350,26 +381,276 @@ header .menu-bars:hover {
   margin: 1em 0.5em;
   font-size: 0.8em;
   color: #fff;
+  font-size: 0.9em;
 }
 
 form {
   margin: 2em 0 1em 0;
   display: flex;
-  justify-content: center;
   align-items: center;
   flex-direction: column;
 }
 
 .contact-input {
   width: 100%;
-  font-size: 1em;
+  font-size: 0.8em;
   padding: 1em;
-  margin-bottom: 0.5em;
+  margin-bottom: 1em;
   border: none;
   border-radius: 0.5em;
   color: #172b4d;
 }
 
+form textarea {
+  color: #b3bac5 !important;
+}
+
 #contact .btn {
   align-self: flex-start;
 }
+
+/* -------------------- Break Points for Tablet & Desktop Layout -------------------- */
+@media only screen and (min-width: 768px) {
+  /* ========== Tablet Styles ========== */
+
+  /* ------------ General Styles ---------- */
+  .btn {
+    width: 40%;
+  }
+
+  /* ------------ Header Styles ---------- */
+
+  header .logo-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 20%;
+  }
+
+  header .menu-bars {
+    display: none;
+  }
+
+  header nav {
+    width: 40%;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  header nav ul {
+    width: 100%;
+    display: flex;
```

Comment:
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.